src/types/app-client.ts

```diff
@@ -192,8 +191,7 @@ export interface AppClientDeployParams extends AppClientDeployCoreParams, AppCli
   schema?: Partial<AppStorageSchema>
 }
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AppClientCallRawArgs extends RawAppCallArgs {}
+export type AppClientCallRawArgs = RawAppCallArgs
```

Comment:
I'm wondering if we revert this back, so we're consistently exporting interfaces.